js/components/MoodCheck/MoodCheckContainer.js

```diff
@@ -0,0 +1,11 @@
+import React, { Component } from "react";
```

Comment:
Components do not contain a container file. Remove this and follow the R10 conventions.